pkg/generators/openapi.go

```diff
@@ -513,6 +515,19 @@ func (g openAPITypeWriter) validatePatchTags(m *types.Member, parent *types.Type
 	return nil
 }
 
+func (g openAPITypeWriter) generateDefault(m *types.Member) error {
+	tag, err := getSingleTagsValue(m.CommentLines, tagDefault)
+	if tag == "" {
+		return err
+	}
+	var i interface{}
+	if err := json.Unmarshal([]byte(tag), &i); err != nil {
+		return fmt.Errorf("failed to unmarshal default: %v", err)
```

Comment:
nit: also include the member name and tag that failed to parse in error? Would be great if the type name the member is from could be included too.

Reply:
Good point, I'll add that.

Reply:
Just ran a test to see what it looks like:
```
2020/10/16 09:39:26 OpenAPI code generation error: Failed executing generator: some packages had errors:
failed to generate default: failed to unmarshal default: invalid character 'd' in literal false (expecting 'a')
```

That's not great, let me see what I can do.

Reply:
Actually, I missed the line before that, which indicates the name of the field. More context can still be added, runs like this now:
```
E1016 09:47:40.459671   78052 openapi.go:309] Error when generating: sub, sub *k8s.io/kube-openapi/test/integration/testdata/defaults.SubStruct
2020/10/16 09:47:40 OpenAPI code generation error: Failed executing generator: some packages had errors:
failed to generate default in k8s.io/kube-openapi/test/integration/testdata/defaults.Defaulted: failed to unmarshal default: invalid character 'd' in literal false (expecting 'a')
```